portal_spa/src/components/custom/SelectAttributes.tsx

```diff
@@ -1,163 +1,231 @@
-                  <SelectValue placeholder="Select attribute" />
-                </SelectTrigger>
-              </FormControl>
-              <SelectContent>
-                {credentials
-                  .find(
-                    (c) =>
-                      String(c.id) ===
-                      form
-                        .watch(`attributes.${index}.credential_id`)
-                        ?.toString()
-                  )
-                  ?.attributes.map((attr) => (
-                    <SelectItem key={attr.id} value={attr.name_en}>
-                      {attr.name_en}
-                    </SelectItem>
-                  ))}
-              </SelectContent>
-            </Select>
-            <FormMessage />
-          </FormItem>
-        )}
-      />
-
-      <FormLabel className="font-medium">
-        Reason{" "}
-        <Tooltip>
-          <TooltipTrigger asChild>
-            <div className="ml-1 inline-flex items-center justify-center w-4 h-4 rounded-full bg-gray-200 text-gray-600 cursor-pointer">
-              <Info className="w-3 h-3" />
+          ) : (
+            <div className="relative">
+              <Input
+                disabled
+                placeholder="Select a credential first"
+                value=""
+                className="h-9 px-3 text-sm font-normal"
+              />
             </div>
-          </TooltipTrigger>
-          <TooltipContent side="top">
-            <p className="text-sm">
-              Please specify the purpose for which you are disclosing this
-              attribute. Be specific — e.g., “age verification for online
-              purchase” or “access to restricted content.”{" "}
-            </p>
-          </TooltipContent>
-        </Tooltip>
-      </FormLabel>
-      <div className="grid md:grid-cols-2 gap-2">
-        <FormField
-          control={control}
-          name={`attributes.${index}.reason_en`}
-          render={({ field }) => (
-            <FormItem>
-              <FormLabel className="text-sm">English</FormLabel>
-              <FormControl>
-                <Input {...field} />
-              </FormControl>
-              <FormMessage />
-            </FormItem>
-          )}
-        />
-        <FormField
-          control={control}
-          name={`attributes.${index}.reason_nl`}
-          render={({ field }) => (
-            <FormItem>
-              <FormLabel className="text-sm">Dutch</FormLabel>
-              <FormControl>
-                <Input {...field} />
-              </FormControl>
-              <FormMessage />
-            </FormItem>
           )}
-        />
+        </div>
+      </div>
+
+      {/* Reason Fields */}
+      <div className="space-y-4">
+        <div className="grid grid-cols-1 lg:grid-cols-2 gap-4">
+          <div className="space-y-2">
+            <Label className="text-sm font-medium">Reason (English)</Label>
+            <Textarea
+              value={value?.reason_en ?? ""}
```

Comment:
Nullish coalescing operator (`??`) may be omitted because `undefined` is a possible value for the `value` attribute. Strangely enough, the value is not allowed to be `null` 🤔 

```suggestion
              value={value?.reason_en}
```